### PR TITLE
[FEATURE] add option to search also isotopic peak before the annotated

### PR DESCRIPTION
**Pull request by timosachsenberg**
Compensates for monoisotopic peak misassignments

### CODE REVIEW

File: src/utils/RNPxlSearch.cpp [hunk 1]
```diff
@@ -1999,7 +1999,7 @@ class RNPxlSearch :
         double precursor_mz = precursor[0].getMZ();
 
         // map (corrected) precursor mass to spectra
-        for (auto i : precursor_isotopes)
+        for (int i : precursor_isotopes)
```
Comment (hroest) [inline src/utils/RNPxlSearch.cpp]: so why does this fail? is this C++14? I thought that should work in C++11 ?

Comment (timosachsenberg) [inline src/utils/RNPxlSearch.cpp]: I changed it to make it easier readable. Auto is great for long typenames but not really needed for ints

**Comment by enetz** [comment]
Does the precursor error calculation consider this? And should this correction be noted somewhere in the output? May be nice to know for some post-processing analyses.

**Comment by timosachsenberg** [comment]
Good point. Currently it behaves like when other search engines do the same. E.g. Msgf will also have a large precursor error.

**Comment by timosachsenberg** [comment]
We can merge this and think about representing isotopic misassignments later. That discussion will probably affect more tools

File: src/utils/RNPxlSearch.cpp (hunk 1 above)
Comment (hroest) [inline src/utils/RNPxlSearch.cpp]: ah ok I thought it caused travis to fail...my bad

**Pull request merged**